data/json/items/generic.json

```diff
@@ -107,6 +107,226 @@
     "to_hit": -2,
     "flags": [ "TRADER_AVOID" ]
   },
+  {
+    "type": "GENERIC",
+    "abstract": "robot_module_abstract",
+    "name": "module template",
+    "description": "This is a template for robot module.  If found in a game it is a bug.",
+    "symbol": "§",
+    "color": "cyan",
+    "material": [ "plastic", "steel" ],
+    "weight": 250,
+    "volume": 2,
+    "price": 32000,
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "targeting_module",
+    "name": "targeting module",
+    "description": "This module integrate visual and proprioceptive information from peripheric sensors and outputs information necessary for accurate aiming.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "identification_module",
+    "name": "identification module",
+    "description": "This module continuously runs image recogntion algorithms to identify friends from foe.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "pathfinding_module",
+    "name": "pathfinding module",
+    "description": "This module uses a combination of vector integration and egocentric mapping to find the best path available.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "memory_module",
+    "name": "memory banks",
+    "description": "Allows for storage and recovery of information.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "sensor_module",
+    "name": "sensor array",
+    "description": "A wide range of sensors meant to give the ability to perceive the surrounding world.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "self_monitoring_module",
+    "name": "self monitoring sensors",
+    "description": "A array of sensors and diagnostic modules allowing the robot to perceive itslef.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "central_module",
+    "name": "central integrating module",
+    "description": "This module is responsible for decision making, it's basically the seige of the AI of the robot.",
```

Comment:
Seige?

Reply:
siege

Reply:
"The siege of the AI?" Where are you getting that? Wikipedia's glossary of AI terms doesn't list "siege" and its English definition doesn't make sense when used there.

Reply:
Alright no problem, I'm not married to that description. I meant the place that holds the AI. I'll accept any better description to replace this one. Unless even the notion itself of a thing that holds an AI makes no sense.
 Also its fine to point out that it makes no sense in english but keep in mind that lots of contributor don't have english as their first language.

Reply:
**TL,DR:** I'd change it to something like -- "it's basically the [home/seat/substrate/...] of the robot's AI." Or, "it's where the robot's AI lives." (Did you mean "seat" the first time?)

**Reasons**
I think this description might still cause confusion for new players, especially regarding what they can do with this part. And if not, there's still some awkwardness in the phrasing.

It sounds like you're getting at something similar to a CPU (central processing unit), right? I assume you don't want to call it a CPU because that'd imply that future AI tech works just like computers today. But it's still the right analogy. CPU = the brain of the computer = does the  processing and makes the decisions. 

Am I grokking you right? If so, I'd actually say the place that runs vs that holds the AI would be the same. The human brain both holds and runs its intelligence. (But then has a bunch of other parts, including memory banks, that are for other functions than just intelligence/decision-making.)

BTW, good work, these are exciting additions!